eve/worker/dockerfile/actions.Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:bionic
+
+ENV HOME_BUILDBOT /var/lib/buildbot
+
+#
+# Install packages needed by the buildchain
+#
+
+RUN apt-get --assume-yes update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends --assume-yes \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    libssl-dev \
+    openssh-client \
+    python \
+    python3 \
+    python3-dev \
+    python3-pip \
+    python3-pkg-resources \
+    python3-setuptools \
+    python-dev \
+    python-pip \
+    python-pkg-resources \
+    python-setuptools \
+    sudo \
+    tox \
+    wget
+
+RUN sudo apt-get install -y software-properties-common \
+    && sudo apt-get update \
+    && sudo add-apt-repository -y ppa:git-core/ppa \
+    && sudo apt-get update \
+    && sudo apt-get install -y git
+
+COPY requirements.txt /tmp/
+
+RUN pip3 install --upgrade pip
+RUN pip3 install -r /tmp/requirements.txt
+
+# Install helm
+RUN curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > /tmp/get_helm.sh && bash /tmp/get_helm.sh
+
+RUN git config --global credential.helper store
+
+#
+# Add user eve
+#
+
+RUN adduser -u 1042 --home /home/eve --disabled-password --gecos "" eve \
+ && adduser eve sudo \
+ && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+#
+# Eve configuration
+#
+
+USER eve
+
+RUN mkdir -p /home/eve \
+    && mkdir -p /home/eve/.ssh/ \
+    && /bin/echo -e "Host bitbucket.org\n\tStrictHostKeyChecking no\n" >> /home/eve/.ssh/config
+
+ENV LANG=C.UTF-8 \
+    BUILD_DISTRO=trusty
+
+#
+# Run buildbot-worker on startup
+#
+
+ARG BUILDBOT_VERSION
+RUN sudo pip2 install buildbot-worker==$BUILDBOT_VERSION
+
+RUN sudo mkdir -p /__w/bert-e/bert-e
+RUN ln -s /__w/bert-e/bert-e /home/eve/workspace
+WORKDIR /home/eve/workspace
+CMD buildbot-worker create-worker . "$BUILDMASTER:$BUILDMASTER_PORT" "$WORKERNAME" "$WORKERPASS" \
+    && buildbot-worker start --nodaemon
```

Comment:
For me you either:
* add the modifications so that `eve/worker/dockerfile/Dockerfile` can also work with actions
* Or you move this file into the `.github` folder and remove every single reference to buildbot and make it very "actions" specific.